homeassistant/components/onvif/parsers.py

```diff
@@ -20,6 +20,11 @@
 }
 
 
+def extract_message(msg: Any) -> tuple[str, Any]:
```

Comment:
Is is possible to replace Any with the correct type in the return tuple?

Reply:
Sadly the whole typing is lacking in the onvif library, so without more knowledge, I can't